bolt-sidecar/src/config.rs

```diff
@@ -27,6 +27,9 @@ pub struct Opts {
     /// Max commitments to accept per block
     #[clap(short = 'm', long)]
     pub(super) max_commitments: Option<usize>,
+    /// Validator indexes
+    #[clap(short = 'v', long, value_parser, num_args = 0.., value_delimiter = ',')]
```

Comment:
Shouldn't `num_args` be `1..` instead of `0..`? In order for Bolt to function properly you must specify at least one validator index

Reply:
good catch, done